platform/linux-generic/arch/aarch64/odp/api/abi/sync_inlines.h

```diff
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2023 Nokia
+ */
+
+#ifndef ODP_ARCH_SYNC_INLINES_H_
+#define ODP_ARCH_SYNC_INLINES_H_
```

Comment:
Looks like a slightly more common naming scheme would be to use `ODP_API_ABI_SYNC_INLINES_H_`, which better matches the file path, but no strong feelings about this.

Reply:
Keeping ARCH as it's more readable and there's another file with that in the same directory already.